torch_xla/core/dynamo_bridge.py

```diff
@@ -417,11 +417,19 @@ def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
 
   # partition the model and exectue to collect inputs
   supported_ops = XlaOperatorSupport()
-  partitioner = CapabilityBasedPartitioner(xla_model, supported_ops)
+  partitioner = CapabilityBasedPartitioner(
+      xla_model, supported_ops, allows_single_node_partition=True)
```

Comment:
The `allows_single_node_partition=True` flag ensures that the partitioned module (even when it is one large partition) goes into our own `extract_internal` call.